pybandits/simulator.py

```
@@ -120,16 +117,7 @@ class Simulator(PyBanditsBaseModel, ABC):
 
-    if pydantic_version == PYDANTIC_VERSION_1:
-
-        class Config:
-            arbitrary_types_allowed = True
-            allow_population_by_field_name = True
-
-    elif pydantic_version == PYDANTIC_VERSION_2:
-        model_config = {"arbitrary_types_allowed": True, "populate_by_name": True}
-    else:
-        raise ValueError(f"Unsupported pydantic version: {pydantic_version}")
+    model_config = {"arbitrary_types_allowed": True, "populate_by_name": True}
```

Ruff (0.15.12)

[warning] Mutable default value for class attribute

(RUF012)